src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.239.1.mdx

```diff
@@ -0,0 +1,39 @@
+---
+subject: Browser agent
+releaseDate: "2023-09-01”
```

Comment:
i think this might need to be a normal `"` instead of slanted

Reply:
Yup, that was it! Thanks @roadlittledawn